### PR TITLE
[RAPTOR-4726] implement /info endpoint

### PR DESCRIPTION
**Pull request by yakov-g**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

## Summary
Implement `/info` endpoint:
Response is smth like that:
```
{
    "DRUM server": "flask",
    "DRUM version": "1.4.15",
    "code_dir": "/home/yakov.goldberg/workspace/datarobot-user-models/model_templates/inference/python3_sklearn",
    "language": "python",
    "model metadata": {
        "environmentID": "5e8c889607389fe0f466c72d",
        "inferenceModel": {
            "targetName": "MEDV"
        },
        "modelID": "5f1f15a4d6111f01cb7f91fd",
        "name": "drumpush-regression",
        "targetType": "regression",
        "type": "inference",
        "validation": {
            "input": "../../../tests/testdata/boston_housing.csv"
        }
    },
    "predictor": "scikit-learn",
    "target_type": "regression"
}
```

### CODE REVIEW

**Comment by yakov-g** [comment]
jarvis test functional_training_model_templates

**Review by rvorobii — COMMENTED** [review]
in addition to the comments I have left
1. it's hard to follow what is the resulting `model_info` response - I think we should `ModelInfo` entity so that you can see a full set of fields in it (although some of them can be optional and missing)
2. we should consider adding capabilities to the `model_info` (csv, arrow etc)

**Review by rvorobii — APPROVED** [review]
we should consider adding capabilities to the model_info (csv, arrow etc)

**Comment by yakov-g** [comment]
jarvis test

**Comment by yakov-g** [comment]
jarvis test functional_training_model_templates

**Comment by yakov-g** [comment]
jarvis test functional_training_model_templates

**Comment by yakov-g** [comment]
jarvis test functional_training_model_templates

**Review by meltzg — APPROVED** [review]
